.github/ISSUE_TEMPLATE/suggestion.md

```diff
@@ -0,0 +1,11 @@
+---
+name: Suggestion
+about: Suggest something to enhance Skript
+
+---
+
+### Description
+<!--- A clear and concise description of the suggestion --->
+
+### Additional Context
```

Comment:
I don't think suggestions need two sections. Usually they already contain enough information to decide whether they should be implemented or not.